### PR TITLE
Update imdex.md for oVirt 4.4.0 rc2

### PR DESCRIPTION
**Pull request by lveyde**
Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

File: source/release/4.4.0/index.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ h1, h2, h3, h4, h5, h6, li, a, p {
...
 
 oVirt is a free open-source distributed virtualization solution,
 designed to manage your entire enterprise infrastructure.
 oVirt uses the trusted KVM hypervisor and is built upon several other community
 projects, including libvirt, Gluster, PatternFly, and Ansible.
 
-This release is available now for Red Hat Enterprise Linux 8.1 and
-CentOS Linux 8.1 (or similar).
+This release is available now for Red Hat Enterprise Linux 7.7 and
+CentOS Linux 7.7 (or similar).
```
Comment (sandrobonazzola) [inline source/release/4.4.0/index.md:24]: 8.1

File: source/release/4.4.0/index.md [hunk 2]
```diff
@@ -13,15 +13,15 @@ h1, h2, h3, h4, h5, h6, li, a, p {
...
+The oVirt Project is pleased to announce the availability of the 4.4.0 Second Release Candidate as of May 14, 2020.
 
 oVirt is a free open-source distributed virtualization solution,
 designed to manage your entire enterprise infrastructure.
 oVirt uses the trusted KVM hypervisor and is built upon several other community
 projects, including libvirt, Gluster, PatternFly, and Ansible.
 
-This release is available now for Red Hat Enterprise Linux 8.1 and
-CentOS Linux 8.1 (or similar).
+This release is available now for Red Hat Enterprise Linux 7.7 and
```
Comment (sandrobonazzola) [inline source/release/4.4.0/index.md:23]: 8.1

Comment (lveyde) [reply source/release/4.4.0/index.md]: Updated.

File: source/release/4.4.0/index.md (hunk 1 above)
Comment (lveyde) [reply source/release/4.4.0/index.md]: Updated.

**Pull request merged**